ts/components/messages/CalendarEventButton.tsx

```diff
@@ -79,17 +78,19 @@ const disabledStyles = StyleSheet.create({
   }
 });
 
+const screenWidth = Dimensions.get("window").width;
+const minScreenWidth = 320;
+
 class CalendarEventButton extends React.PureComponent<Props> {
   public render() {
-    const {
-      isEventInDeviceCalendar,
-      small,
-      disabled,
-      onPress,
-      useShortLabel
-    } = this.props;
-
+    const { isEventInDeviceCalendar, small, disabled, onPress } = this.props;
     const iconName = isEventInDeviceCalendar ? "io-tick-big" : "io-plus";
+    // On small devices use short label
+    const reminderText = I18n.t(
```

Comment:
It could be placed outside of the render method (and the class too)
Since it doesn't depends on any field class or method you can place it outside the class as const
and avoid the comparison (`screenWidth <= minScreenWidth`) is done each time render is called

Reply:
:+1: :man_facepalming: :man_facepalming: :man_facepalming: